### ISSUE TITLE
Tested on ESP32?

### ISSUE THREAD

**Issue by chadham** [martin-ger/esp_mqtt#20]
Fantastic work - game changer for IOT on MQTT!
Have you tested this on the ESP32 by any chance? It has considerable more system resources so could be more viable. 

Thanks.

**Comment by martin-ger** [comment]
No, not yet. Porting is some effort as tasking is different under RTOS and the socket interface also.

Think about it - if I find some time...

**Comment by chadham** [comment]
Okay. Let me know if I can help test it...

**Comment by Fbo06** [comment]
Hello, first thank you very much for this fantastic work, do you have any news about porting this library to ESP32, because it is much much more powerful ?

**Comment by 1003657663** [comment]
so cool.  if this can support esp32 is really cool.
